python/eos/analysis_file.py

```diff
@@ -104,8 +104,12 @@ def __init__(self, analysis_file):
                     qn = eos.QualifiedName(name)
                     unit = eos.Unit(param['unit'])
                     central, min, max = float(param['central']), float(param['min']), float(param['max'])
-                    eos.Parameters.declare(qn, param['latex'], unit, central, min, max)
+                    id = eos.Parameters.declare(qn, param['latex'], unit, central, min, max)
                     eos.info(f'Successfully declared parameter: {qn}')
+                    if 'alias_of' in param:
```

Comment:
``'alias_of'`` still needs to be recognized as an optional key to avoid the warning message in line 100.

Reply:
I fixed it (and used sets instead of lists).